docker/genetics/Dockerfile

```diff
@@ -5,6 +5,9 @@ RUN hail-apt-get-install \
     libbz2-dev \
     libcurl4-openssl-dev \
     liblzma-dev \
+    libvcflib-tools \
```

Comment:
Why did you add these?

Reply:
Compiling VCFlib was hard and they now have an apt-get package